src/modules/account/internal/store/account-repository.ts

```diff
@@ -0,0 +1,37 @@
+import mongoose from 'mongoose';
+import ConfigService from '../../../config/config-service';
+import AccountDB from '../account-db';
+import accountSchema from './account-schema';
+
+export default class AccountRepository {
+  private static connection: mongoose.Connection;
+
+  private static account: mongoose.Model<mongoose.Document>;
+
+  static initDb(): void {
+    const mongoURI = ConfigService.getStringValue('mongoDb.uri');
+    AccountRepository.connection = mongoose.createConnection(mongoURI);
+    AccountRepository.account = AccountRepository.connection.model(
+      'Account',
+      accountSchema,
+    );
+  }
+
+  static async findOne(
```

Comment:
We should avoid this. Reader class should directly `AccountDB.findOne()` or something to that effect. It's ok to do init at bootup up of service but otherwise we should not use this